x-pack/test_serverless/functional/test_suites/search/search_synonyms/search_synonyms_overview.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const pageObjects = getPageObjects([
+    'svlCommonPage',
+    'svlCommonNavigation',
+    'searchSynonyms',
+    'embeddedConsole',
+    'common',
+  ]);
+  const browser = getService('browser');
+  const es = getService('es');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('Serverless Synonyms Overview', function () {
+    before(async () => {
+      await kibanaServer.uiSettings.update({ 'searchSynonyms:synonymsEnabled': 'true' });
+      await pageObjects.svlCommonPage.loginWithRole('admin');
```

Comment:
this should probably `developer` instead but we can fix that later.

Reply:
I will include multiple tests